<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeaponSelect</class>
 <widget class="QWidget" name="WeaponSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>3</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪文黑-85W</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">/*按钮普通态*/
QPushButton
{
	/*字体为微软雅黑*/
    font-family:汉仪文黑-85W;
    /*字体大小为20点*/
    font-size:25pt;
    /*字体颜色为黑色*/    
    color:black;
    /*背景颜色*/  
    background-color:transparent;
    /*边框圆角半径为25像素*/ 
    border-radius:21px;
	border-image: url(:/Icon/close.png)
}

/*按钮停留态*/
/*QPushButton:hover*/
/*{*/
    /*背景颜色*/  
    /*background-color: #ecf5ff;*/
	/*字体颜色*/  
    /*color: #409eff;*/
/*}*/

/*按钮按下态*/
QPushButton:pressed
{
    /*字体颜色为黑色*/    
    color:black;
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
	border-image: url(:/Icon/close_selected.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="text">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>9</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪文黑-85W</family>
     <pointsize>10</pointsize>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(211, 188, 142);</string>
   </property>
   <property name="text">
    <string>选择武器</string>
   </property>
  </widget>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>540</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-image: url(:/BG/background.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bgWeap">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>540</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QScrollArea" name="weapArea">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>420</width>
     <height>420</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="weapon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>420</width>
      <height>420</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <widget class="QScrollBar" name="verticalScrollBar_3">
     <property name="geometry">
      <rect>
       <x>416</x>
       <y>0</y>
       <width>4</width>
       <height>420</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 设置垂直滚动条基本样式*/
QScrollBar:vertical
{
    width:4px;
    background:#515151;
    margin:0px,0px,0px,0px;
    padding-top:4px;   /* 留出9px给上面和下面的箭头*/
    padding-bottom:4px;
}
QScrollBar::handle:vertical
{
    width:4px;
    background:white;
    border-radius:4px;   /* 滚动条两端变成椭圆 */
    min-height:20;
}
QScrollBar::handle:vertical:hover
{
    width:4px;
    background:white;   /* 鼠标放到滚动条上的时候，颜色变深*/
    border-radius:4px;
    min-height:20;
}
QScrollBar::add-line:vertical   /* 这个应该是设置下箭头的，3.png就是箭头*/
{
    border:none;
}
QScrollBar::sub-line:vertical   /* 设置上箭头*/
{
    border:none;
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>490</y>
     <width>125</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;修改所选圣遗物属性&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: transparent;
border-radius:31px;
border-image: url(:/Icon/confirm.png)
}

QPushButton:pressed
{
border-image: url(:/Icon/confirm_selected.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QScrollArea" name="detailArea">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>224</width>
     <height>390</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="detail">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>224</width>
      <height>390</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(59, 66, 85, 216);</string>
    </property>
    <widget class="QLabel" name="name">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>3</y>
       <width>170</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="type">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>30</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #dbdbdb;
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ba">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>50</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 25);
padding-left:4px;
border-top-left-radius:2px;
border-bottom-left-radius:2px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="bav">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>50</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 25);
padding-left:4px;
border-top-right-radius:2px;
border-bottom-right-radius:2px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="level">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>125</y>
       <width>75</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 76);
border-radius:2px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="SpecialAbilityName">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>170</y>
       <width>200</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="SpecialAbility">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>195</y>
       <width>205</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>205</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>205</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(255, 255, 255, 153);
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QScrollBar" name="verticalScrollBar_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>4</width>
       <height>420</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 设置垂直滚动条基本样式*/
QScrollBar:vertical
{
    width:4px;
    background:#515151;
    margin:0px,0px,0px,0px;
    padding-top:4px;   /* 留出9px给上面和下面的箭头*/
    padding-bottom:4px;
}
QScrollBar::handle:vertical
{
    width:4px;
    background:white;
    border-radius:4px;   /* 滚动条两端变成椭圆 */
    min-height:20;
}
QScrollBar::handle:vertical:hover
{
    width:4px;
    background:white;   /* 鼠标放到滚动条上的时候，颜色变深*/
    border-radius:4px;
    min-height:20;
}
QScrollBar::add-line:vertical   /* 这个应该是设置下箭头的，3.png就是箭头*/
{
    height:4px;width:6px;
    background-color: transparent;
    border-image:url(:/Icon/down.png);
    subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical   /* 设置上箭头*/
{
    height:4px;width:6px;
	background-color: transparent;
    border-image:url(:/Icon/up.png);
    subcontrol-position:top;
}
QScrollBar::add-line:vertical:hover   /* 当鼠标放到下箭头上的时候*/
{
    height:4px;width:6px;
    background-color: transparent;
    border-image:url(:/Icon/down_selected.png);
    subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical:hover  /* 当鼠标放到下箭头上的时候*/
{
    height:4px;width:6px;
	background-color: transparent;
    border-image:url(:/Icon/up_selected.png);
    subcontrol-position:top;
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="star">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>105</y>
       <width>84</width>
       <height>15</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="des">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>264</y>
       <width>205</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>205</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>205</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(255, 255, 255, 153);
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="mav">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>75</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 25);
padding-left:4px;
border-top-right-radius:2px;
border-bottom-right-radius:2px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="ma">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>75</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 25);
padding-left:4px;
border-top-left-radius:2px;
border-bottom-left-radius:2px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="grade">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>145</y>
       <width>200</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255,233,153);
background-color: transparent;</string>
     </property>
     <property name="text">
      <string>精炼1/2/3/4/5阶</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox
{
  color:#495366;
  font: 12pt &quot;汉仪文黑-85W&quot;;
  background-color: #c0bdb7;
  padding: 1px 15px 1px 3px;
  border:1px solid rgba(228,228,228,1);
  border-radius:5px 5px 0px 0px;
}

QComboBox::drop-down 
{
  background-color: transparent;
}

QComboBox QAbstractItemView
{
	background:#3d4556;
  border:1px solid rgba(228,228,228,1);
  border-radius:0px 0px 5px 5px;
  outline: 0px;
}

QComboBox QAbstractItemView::item
{
	height:31px;
  font: 12pt &quot;汉仪文黑-85W&quot;;
	color:#ece5d8;
	padding-left:9px;
	background-color:#3d4556;
}
QComboBox QAbstractItemView::item:hover
{
  background-color:#e1dbd0;
  color:#495366;
}

QComboBox QAbstractItemView::item:selected
{
  background-color:#e1dbd0;
  color:#495366;
}

/******** combobox 滚动条  *********/
QComboBox QScrollBar::vertical{ /*主体部分*/
  width:4px;
  background:rgb(89,89,89);
  border:none;
  border-radius:5px;
}
QComboBox QScrollBar::handle::vertical{ /*滑块主体*/
  border-radius:5px;
  width: 4px;
  background:rgb(173,173,173);
}
QComboBox QScrollBar::handle::vertical::hover{
  background:#e1dbd0;
}
QComboBox QScrollBar::add-line::vertical{/*上箭头*/
  border:none;
}
QComboBox QScrollBar::sub-line::vertical{/*下箭头*/
  border:none;
}</string>
   </property>
  </widget>
  <zorder>bg</zorder>
  <zorder>bgWeap</zorder>
  <zorder>Back</zorder>
  <zorder>text</zorder>
  <zorder>weapArea</zorder>
  <zorder>confirm</zorder>
  <zorder>detailArea</zorder>
  <zorder>comboBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
